website/src/roadmap/portability.md

```diff
@@ -169,3 +169,6 @@ To run on a separate [Flink cluster](https://ci.apache.org/projects/flink/flink-
 4. Start JobService with Flink web service endpoint: `./gradlew :beam-runners-flink_2.11-job-server:runShadow -PflinkMasterUrl=localhost:8081`
```

Comment:
I think we should remove the instructions here and have them on the Flink runner page only.

Reply:
I thought about that. Instead I kept the wordcount example here and the more programmatic instructions on the Flink Runner page.

Reply:
OK, but the "To run on a separate Flink cluster:" part is too detailed (even with optimization instructions) to belong there?

Reply:
Yes, and these optimization instructions are kind of questionable. I think I'll remove those.